share/completions/apm.fish

```diff
@@ -40,7 +36,11 @@ function __fish_apm_using_command
                         return 1
                     end
                 case '*'
-                    return 1
+                    if test $command_seen_once -eq 0
+                        continue
```

Comment:
Why continue? Why not "return 0"? You've seen a command, the next word is presumably the argument for that command.